diff -urN courier-authlib-0.59.orig/Makefile.am courier-authlib-0.59/Makefile.am
--- courier-authlib-0.59.orig/Makefile.am	2007-01-08 13:18:42.897900250 +0100
+++ courier-authlib-0.59/Makefile.am	2007-01-08 13:20:17.587818000 +0100
@@ -6,8 +6,8 @@
 AUTOMAKE = srcdir=${srcdir} @SHELL@ ${srcdir}/automake.fix @AUTOMAKE@
 AUTOMAKE_OPTIONS=dist-bzip2
 
-SUBDIRS=@LIBLTDL_SUBDIR@ @DBSUBDIR@ md5 sha1 libhmac numlib makedat userdb rfc822 random128 liblock liblog
-DIST_SUBDIRS=libltdl gdbmobj bdbobj md5 sha1 libhmac numlib makedat userdb rfc822 random128 liblock liblog
+SUBDIRS=@DBSUBDIR@ md5 sha1 libhmac numlib makedat userdb rfc822 random128 liblock liblog
+DIST_SUBDIRS=gdbmobj bdbobj md5 sha1 libhmac numlib makedat userdb rfc822 random128 liblock liblog
 
 INCLUDES = $(LTDLINCL)
 
diff -urN courier-authlib-0.59.orig/configure.in courier-authlib-0.59/configure.in
--- courier-authlib-0.59.orig/configure.in	2007-01-08 13:18:42.897900250 +0100
+++ courier-authlib-0.59/configure.in	2007-01-08 13:18:58.318864000 +0100
@@ -1083,7 +1083,7 @@
 AC_SUBST(REPOSITORY)
 
 AM_CONDITIONAL(HAVE_SGML, test -d ${srcdir}/docbook)
-AC_CONFIG_SUBDIRS(libltdl bdbobj gdbmobj md5 sha1 libhmac numlib makedat userdb rfc822 random128 liblock liblog)
+AC_CONFIG_SUBDIRS(bdbobj gdbmobj md5 sha1 libhmac numlib makedat userdb rfc822 random128 liblock liblog)
 
 AC_CONFIG_FILES(Makefile authdaemond authdaemonrc authsystem.passwd README.authdebug.html dbobj.h authmigrate courier-authlib.spec courier-authlib.lpspec courier-authlib.sysvinit userdb-test-cram-md5.pl)
 AC_OUTPUT
