--- courier-authlib-0.68.0.orig/libs/bdbobj/Makefile.am	2008-05-08 18:00:52.000000000 +0200
+++ courier-authlib-0.68.0/libs/bdbobj/Makefile.am	2008-05-29 15:33:05.000000000 +0200
@@ -20,7 +20,6 @@
 
 testbdb_DEPENDENCIES=libbdbobj.la
 testbdb_LDADD=libbdbobj.la
-testbdb_LDFLAGS=-static
 
 check-am:
 	$(srcdir)/testsuite | cmp -s - $(srcdir)/testsuite.txt
--- courier-authlib-0.68.0.orig/libs/gdbmobj/Makefile.am	2008-05-08 18:00:52.000000000 +0200
+++ courier-authlib-0.68.0/libs/gdbmobj/Makefile.am	2008-05-29 15:33:36.000000000 +0200
@@ -20,7 +20,6 @@
 
 testgdbm_DEPENDENCIES=libgdbmobj.la
 testgdbm_LDADD=libgdbmobj.la
-testgdbm_LDFLAGS=-static
 
 check-am:
 	$(srcdir)/testsuite | cmp -s - $(srcdir)/testsuite.txt
--- courier-authlib-0.68.0.orig/libs/libhmac/Makefile.am	2007-07-21 22:12:29.000000000 +0200
+++ courier-authlib-0.68.0/libs/libhmac/Makefile.am	2008-05-29 15:33:27.000000000 +0200
@@ -15,12 +15,10 @@
 md5hmactest_SOURCES=md5hmactest.c
 md5hmactest_DEPENDENCIES=libhmac.la @hashlibs@
 md5hmactest_LDADD=$(md5hmactest_DEPENDENCIES)
-md5hmactest_LDFLAGS=-static
 
 sha1hmactest_SOURCES=sha1hmactest.c
 sha1hmactest_DEPENDENCIES=libhmac.la @hashlibs@
 sha1hmactest_LDADD=$(sha1hmactest_DEPENDENCIES)
-sha1hmactest_LDFLAGS=-static
 
 EXTRA_DIST=md5hmactest.txt sha1hmactest.txt
 
--- courier-authlib-0.68.0.orig/libs/liblock/Makefile.am	2007-07-21 22:12:29.000000000 +0200
+++ courier-authlib-0.68.0/libs/liblock/Makefile.am	2008-05-29 15:33:02.000000000 +0200
@@ -32,7 +32,6 @@
 lockmail_SOURCES=lockmail.c
 lockmail_DEPENDENCIES=liblock.la ../numlib/libnumlib.la
 lockmail_LDADD=liblock.la ../numlib/libnumlib.la
-lockmail_LDFLAGS=-static
 
 noinst_DATA=lockmail.html lockmail.1
 EXTRA_DIST=lockfcntl.c lockflock.c locklockf.c locktest.c $(noinst_DATA)
--- courier-authlib-0.68.0/liblog/Makefile.am.orig	2018-03-18 18:44:52.505942103 +0100
+++ courier-authlib-0.68.0/liblog/Makefile.am	2018-03-18 18:45:27.955941698 +0100
@@ -12,7 +12,6 @@
 courierlogger_SOURCES=logger.c
 courierlogger_DEPENDENCIES=../libs/liblock/liblock.la ../libs/numlib/libnumlib.la
 courierlogger_LDADD=../libs/liblock/liblock.la ../libs/numlib/libnumlib.la
-courierlogger_LDFLAGS=-static
 
 if HAVE_SGML
 courierlogger.html: courierlogger.sgml ../libs/docbook/sgml2html
--- courier-authlib-0.68.0.orig/libs/makedat/Makefile.am	2008-05-08 18:00:53.000000000 +0200
+++ courier-authlib-0.68.0/libs/makedat/Makefile.am	2008-05-29 15:33:21.000000000 +0200
@@ -11,4 +11,3 @@
 makedatprog_SOURCES=makedatprog.c
 makedatprog_DEPENDENCIES=@dblibrary@
 makedatprog_LDADD=@dblibrary@
-makedatprog_LDFLAGS=-static
--- courier-authlib-0.68.0.orig/Makefile.am	2008-05-08 19:20:38.000000000 +0200
+++ courier-authlib-0.68.0/Makefile.am	2008-05-29 15:32:56.000000000 +0200
@@ -346,7 +346,6 @@
 authdaemontest_SOURCES=authdaemontest.c
 authdaemontest_DEPENDENCIES=libcourierauthcommon.la
 authdaemontest_LDADD=libcourierauthcommon.la @NETLIBS@
-authdaemontest_LDFLAGS=-static
 
 authmksock_SOURCES=authmksock.c
 authmksock_LDADD=@NETLIBS@
--- courier-authlib-0.68.0.orig/libs/md5/Makefile.am	2007-07-21 22:12:29.000000000 +0200
+++ courier-authlib-0.68.0/libs/md5/Makefile.am	2008-05-29 15:32:59.000000000 +0200
@@ -21,7 +21,6 @@
 md5test_SOURCES=md5test.c
 md5test_DEPENDENCIES=libmd5.la
 md5test_LDADD=libmd5.la
-md5test_LDFLAGS=-static
 
 EXTRA_DIST=md5test.txt hmac.c
 
--- courier-authlib-0.68.0/libs/rfc822/Makefile.am.orig	2018-03-18 18:47:57.425939991 +0100
+++ courier-authlib-0.68.0/libs/rfc822/Makefile.am	2018-03-18 18:49:04.825939219 +0100
@@ -25,12 +25,10 @@
 testsuite_SOURCES=testsuite.c
 testsuite_DEPENDENCIES=librfc822.la
 testsuite_LDADD=librfc822.la -lcourier-unicode
-testsuite_LDFLAGS=-static
 
 reftest_SOURCES=reftest.c imaprefs.h
 reftest_DEPENDENCIES=librfc822.la
 reftest_LDADD=librfc822.la -lcourier-unicode
-reftest_LDFLAGS=-static
 
 EXTRA_DIST=testsuite.txt reftest.txt $(BUILT_SOURCES)
 
--- courier-authlib-0.68.0.orig/libs/sha1/Makefile.am	2007-07-21 22:12:29.000000000 +0200
+++ courier-authlib-0.68.0/libs/sha1/Makefile.am	2008-05-29 15:33:33.000000000 +0200
@@ -20,7 +20,6 @@
 testsuite_SOURCES=testsuite.c
 testsuite_DEPENDENCIES=libsha1.la
 testsuite_LDADD=$(testsuite_DEPENDENCIES)
-testsuite_LDFLAGS=-static
 
 EXTRA_DIST=testsuite.txt hmac.c
 
--- courier-authlib-0.71.4/userdb/Makefile.am.orig	2022-05-21 22:11:47.387647448 +0200
+++ courier-authlib-0.71.4/userdb/Makefile.am	2022-05-21 22:12:28.227426200 +0200
@@ -16,7 +16,6 @@ noinst_DATA=makeuserdb.html userdb.html
 
 userdbpw_SOURCES=userdbpw.c
 userdbpw_LDADD=libuserdb.la @HMACLIB@ @MD5LIB@ @SHA1LIB@ @CRYPTLIBS@
-userdbpw_LDFLAGS=-static
 
 man8=makeuserdb.8 userdb.8 userdbpw.8 pw2userdb.8
 man_MANS=$(man8)
