--- courier-authlib-0.58/gdbmobj/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/gdbmobj/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -20,7 +20,6 @@
 
 testgdbm_DEPENDENCIES=libgdbmobj.la
 testgdbm_LDADD=libgdbmobj.la @LIBGDBM@
-testgdbm_LDFLAGS=-static
 
 check-am:
 	$(srcdir)/testsuite | cmp -s - $(srcdir)/testsuite.txt
--- courier-authlib-0.58/bdbobj/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/bdbobj/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -20,7 +20,6 @@
 
 testbdb_DEPENDENCIES=libbdbobj.la
 testbdb_LDADD=libbdbobj.la @LIBDB@
-testbdb_LDFLAGS=-static
 
 check-am:
 	$(srcdir)/testsuite | cmp -s - $(srcdir)/testsuite.txt
--- courier-authlib-0.58/md5/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/md5/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -22,7 +22,6 @@
 md5test_SOURCES=md5test.c
 md5test_DEPENDENCIES=libmd5.la
 md5test_LDADD=libmd5.la
-md5test_LDFLAGS=-static
 
 EXTRA_DIST=md5test.txt hmac.c
 
--- courier-authlib-0.58/sha1/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/sha1/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -21,7 +21,6 @@
 testsuite_SOURCES=testsuite.c
 testsuite_DEPENDENCIES=libsha1.la
 testsuite_LDADD=$(testsuite_DEPENDENCIES)
-testsuite_LDFLAGS=-static
 
 EXTRA_DIST=testsuite.txt hmac.c
 
--- courier-authlib-0.58/libhmac/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/libhmac/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -16,12 +16,10 @@
 md5hmactest_SOURCES=md5hmactest.c
 md5hmactest_DEPENDENCIES=libhmac.la @hashlibs@
 md5hmactest_LDADD=$(md5hmactest_DEPENDENCIES)
-md5hmactest_LDFLAGS=-static
 
 sha1hmactest_SOURCES=sha1hmactest.c
 sha1hmactest_DEPENDENCIES=libhmac.la @hashlibs@
 sha1hmactest_LDADD=$(sha1hmactest_DEPENDENCIES)
-sha1hmactest_LDFLAGS=-static
 
 EXTRA_DIST=md5hmactest.txt sha1hmactest.txt
 
--- courier-authlib-0.58/makedat/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/makedat/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -12,4 +12,3 @@
 makedatprog_SOURCES=makedatprog.c
 makedatprog_DEPENDENCIES=@dblibrary@
 makedatprog_LDADD=@dblibrary@ @LIBGDBM@ @LIBDB@
-makedatprog_LDFLAGS=-static
--- courier-authlib-0.58/userdb/Makefile.am~	2005-07-16 21:25:25.000000000 +0300
+++ courier-authlib-0.58/userdb/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -19,7 +19,6 @@
 
 userdbpw_SOURCES=userdbpw.c
 userdbpw_LDADD=libuserdb.la @HMACLIB@ @MD5LIB@ @SHA1LIB@ @CRYPTLIBS@
-userdbpw_LDFLAGS=-static
 
 man8=makeuserdb.8 userdb.8 userdbpw.8
 man_MANS=$(man8)
--- courier-authlib-0.58/Makefile.am~	2005-08-11 02:52:58.000000000 +0300
+++ courier-authlib-0.58/Makefile.am	2006-11-09 22:14:18.548842833 +0200
@@ -357,7 +357,6 @@
 authdaemontest_SOURCES=authdaemontest.c
 authdaemontest_DEPENDENCIES=libcourierauthcommon.la
 authdaemontest_LDADD=libcourierauthcommon.la @NETLIBS@
-authdaemontest_LDFLAGS=-static
 
 authmksock_SOURCES=authmksock.c
 authmksock_LDADD=@NETLIBS@
