diff -Nur old/configure.in new/configure.in
--- old/configure.in	2004-12-05 02:15:54.000000000 +0000
+++ new/configure.in	2004-12-10 22:40:07.493270888 +0000
@@ -885,7 +885,7 @@
 then
 	$COURIERCONFIG >conftest.out || exit 1
 	sed -n '/^mail/p' <conftest.out >conftest2.out || exit 1
-	. conftest2.out
+	. `pwd`/conftest2.out
 	rm -f conftest.out conftest2.out
 
 	cmailuser="$mailuser"
@@ -897,7 +897,7 @@
 then
 	$bindir/courierauthconfig --configfiles >conftest.out || exit 1
 	sed -n '/^mail/p' <conftest.out >conftest2.out || exit 1
-	. conftest2.out
+	. `pwd`/conftest2.out
 	rm -f conftest.out conftest2.out
 
 	cmailuser="$mailuser"
