diff -u courier-authlib-0.65.0.orig/Makefile.am courier-authlib-0.65.0/Makefile.am
--- courier-authlib-0.65.0.orig/Makefile.am	2013-08-04 10:43:06.000000000 +0000
+++ courier-authlib-0.65.0/Makefile.am	2013-08-04 10:46:20.512179254 +0000
@@ -4,7 +4,7 @@
 
 AUTOMAKE_OPTIONS=dist-bzip2
 
-SUBDIRS=gdbmobj bdbobj md5 sha1 libhmac numlib makedat userdb unicode rfc822 random128 liblock liblog
+SUBDIRS=libltdl gdbmobj bdbobj md5 sha1 libhmac numlib makedat userdb unicode rfc822 random128 liblock liblog
 
 INCLUDES = $(LTDLINCL)
 
