--- courier-authlib-0.71.4/checkpasswordmd5.c.orig	2022-05-21 21:47:03.355687139 +0200
+++ courier-authlib-0.71.4/checkpasswordmd5.c	2022-05-21 21:52:29.367254317 +0200
@@ -24,7 +24,11 @@ int authcheckpasswordmd5(const char *pas
 
 	if (strncasecmp(encrypted_password, "{MD5}", 5) == 0)
 	{
-               return (strcmp(encrypted_password+5, md5_hash_courier(password)));
+               int a = strcmp(encrypted_password+5, md5_hash_courier(password));
+               if (a != 0)
+                   return (strcasecmp(encrypted_password+5, md5_hash_raw(password)));
+               else
+                   return(a);
 	}
 	if (strncasecmp(encrypted_password, "{MD5RAW}", 8) == 0)
 	{
