--- courier-authlib-0.68.0/authdaemond.c.orig	2017-01-28 03:57:11.000000000 +0100
+++ courier-authlib-0.68.0/authdaemond.c	2018-03-19 19:12:27.664936669 +0100
@@ -114,7 +114,10 @@
 
 		fprintf(stderr, "INFO: Installing %s\n", buf);
 		p += i;
-		h=lt_dlopenext(buf);
+		sprintf(buf2, "courier_%s_init", buf+3);
+
+		strcat(buf, ".so");
+		h=lt_dlopen(buf);
 
 		if (h == NULL)
 		{
@@ -122,8 +125,6 @@
 			continue;
 		}
 
-		sprintf(buf2, "courier_%s_init", buf+3);
-
 		pt=lt_dlsym(h, buf2);
 		if (pt == NULL)
 		{
